caffe2/python/onnx/tests/onnx_backend_test.py

```diff
@@ -35,7 +35,8 @@
 
 backend_test.exclude(r'(test_hardsigmoid'  # Does not support Hardsigmoid.
                      '|test_mean|test_hardmax'  # Does not support Mean and Hardmax.
-                     '|test_cast.*FLOAT16.*)')  # Does not support Cast on Float16.
+                     '|test_cast.*FLOAT16.*'  # Does not support Cast on Float16.
+                     '|test_averagepool_.*same.*)')  # Does not support Cast on Float16.
```

Comment:
tbh initially when I added these `include` `exclude` thing to the backend test suite I imagined people will have multiple invocations with each only contains one pattern, but turns out you folks are too good at using regular expression 😂